node/nodeRunner.go

```diff
@@ -1345,11 +1341,11 @@ func copySingleFile(folder string, configFile string) {
 }
 
 func indexValidatorsListIfNeeded(
-	elasticIndexer process.Indexer,
+	outportHandler outport.OutportHandler,
 	coordinator sharding.NodesCoordinator,
 	epoch uint32,
 ) {
-	if check.IfNil(elasticIndexer) {
+	if check.IfNil(outportHandler) || !outportHandler.HasDrivers() {
```

Comment:
we should not have a nil outport handler here. Should remove the `check.IfNil(outportHandler)`